apis/clusters/v1beta1/postgresql_types.go

```diff
@@ -98,7 +99,7 @@ type PgStatus struct {
 //+kubebuilder:printcolumn:name="ID",type="string",JSONPath=".status.id"
 //+kubebuilder:printcolumn:name="State",type="string",JSONPath=".status.state"
 
-// PostgreSQL is the Schema for the postgresqls API
+// PostgreSQL is the Schema for the postgresqls API:
```

Comment:
Why did you add `:`. We do not have them in other cases

Reply:
Only in 1 place?

Reply:
fixed